### DISCUSSION TITLE
CE edition support for sharing inner container images?

### DISCUSSION THREAD

**Discussion by cu** [nestybox/sysbox#779] [Q&A]
Hello,

One feature that we are interested in is sharing inner container images across sysbox containers. (This is, predictably, in a CI setting.) In the README, there is a table of sysbox features and lists this as a feature of the sysbox EE, but not CE. However, it seems the EE version no longer exists. And I can find no reference to this feature in the docs. I was wondering if I could bother someone on the team with a few questions:

1. Does this feature already happen to exist in the CE edition and is perhaps just undocumented at this time? (One can hope...)
2. If not, are there plans to bring it to the CE?
3. Are there plans to bring any other EE features to the CE?
4. Are there other ways to share container images among multiple instances of docker?

**Comment by ctalledo** [comment]
Hi @cu, 

Thanks for asking; no, there are no plans to bring this feature into CE at the moment.

Just in case, there's a short description of the feature [here](https://github.com/nestybox/sysbox/blob/master/docs/user-guide/images.md#inner-docker-image-sharing). Out of curiosity, does that description of the feature match your needs?

Thanks!
